pkg/machine/ignition/ignition.go

```diff
@@ -692,3 +711,29 @@ func DefaultReadyUnitFile() parser.UnitFile {
 	u.Add("Install", "RequiredBy", "default.target")
 	return *u
 }
+
+func GetRosettaActivationUnitFile() *parser.UnitFile {
```

Comment:
I would assume  this should be moved into the Containerfile now and not ignition. https://github.com/containers/podman/pull/21778
cc @baude

Reply:
As a first step, I removed processing via ssh. To do this, the return value of rosetta-activation.service was always set to true, so that systemd would not log an error.
If this works, I may next consider porting to Containerfile.

The environment variable may be debatable, but I think someone may want to temporarily disable it in very few cases.

Reply:
@Luap99 Fixed and it have been moved to machine-os.
https://github.com/containers/podman-machine-os/pull/7